src/stories/ProposalListTable.stories.tsx

```diff
@@ -35,7 +35,7 @@ export const Default: Story = {
       organization_mission_statement: 'Organization Mission Statement',
       organization_start_date: 'Organization Start Date',
       organization_operating_budget: 'Organization Operating Budget',
-      proposal_title: 'Proposal Title',
+      proposal_name: 'Proposal Title',
```

Comment:
I think we want to fix the field name too, right? I checked api.pdco and it does have `Proposal Name` as the label.

```suggestion
      proposal_name: 'Proposal Name',
```

Reply:
🤦 Good catch, yes!

Reply:
@jasonaowen Fixed (via a force-push rather than suggestion-acceptance since this was in two files). Requires re-review when you have a moment!